external-api/src/v1/controllers/estore/eStore.controller.ts

```diff
@@ -23,27 +9,37 @@ import { eStoreTermStoreAndBuyerFolder, eStoreSiteCreationCron } from '../../../
 import { createExporterSite, siteExists } from './eStoreApi';
 import { getNowAsEpoch } from '../../../helpers/date';
 
-dotenv.config();
-
-const { ESTORE_CRON_MANAGER_SCHEDULE, TZ } = process.env;
-
 export const create = async (req: Request, res: Response) => {
   try {
+    const { body } = req as {
+      body: object;
+    };
+
     // Ensure `req.body` is valid
-    if (objectIsEmpty(req.body)) {
+    if (objectIsEmpty(body)) {
       return res.status(HttpStatusCode.BadRequest).send({ status: HttpStatusCode.BadRequest, message: 'Invalid request' });
     }
 
     // Ensure new CRON job creation
     const cronJobLogs = await getCollection('cron-job-logs');
     const tfmDeals = await getCollection('tfm-deals');
 
-    const { dealId, siteId, facilityIdentifiers, supportingInformation, exporterName, buyerName, dealIdentifier, destinationMarket, riskMarket } = req.body;
+    const { dealId, siteId, facilityIdentifiers, supportingInformation, exporterName, buyerName, dealIdentifier, destinationMarket, riskMarket } = req.body as {
+      dealId: ObjectId;
+      siteId: string;
+      facilityIdentifiers: number[];
+      supportingInformation: string[];
+      exporterName: string;
+      buyerName: string;
+      dealIdentifier: string;
+      destinationMarket: string;
+      riskMarket: string;
+    };
 
     let eStoreData = {} as Estore;
 
     eStoreData = {
```

Comment:
I think we can massively cut down a bunch of this by the way -- like the below

```suggestion

export const create = async (req: Request<unknown, unknown, Estore>, res: Response) => {
  try {
    const { body } = req;

    // Ensure `req.body` is valid
    if (objectIsEmpty(body)) {
      return res.status(HttpStatusCode.BadRequest).send({ status: HttpStatusCode.BadRequest, message: 'Invalid request' });
    }

    // Ensure new CRON job creation
    const cronJobLogs = await getCollection('cron-job-logs');
    const tfmDeals = await getCollection('tfm-deals');

    const { dealId, siteId, facilityIdentifiers, supportingInformation, exporterName, buyerName, dealIdentifier, destinationMarket, riskMarket } = body;

    const eStoreData: Estore = {
```